roles/example-cnf-catalog/defaults/main.yml

```diff
@@ -6,9 +6,8 @@ registry_url: "jumphost.{{ cluster_name }}.dfwt5g.lab:5000"
 repo_name: nfv-example-cnf
 
 image_pull_policy: Always
-cluster_repo_name: "{{ repo_name }}{% if cluster_name %}-{{ cluster_name }}{% endif %}"
 
 catalog_name: nfv-example-cnf-catalog
-catalog_image: "{{ registry_url }}/{{ cluster_repo_name }}/{{ catalog_name }}:{{ operator_version }}"
+catalog_image: "{{ registry_url }}/{{ repo_name }}/{{ catalog_name }}:{{ operator_version }}"
```

Comment:
`cluster_repo_name` is added to allow the deployment to have different registries for applications and operators. It has been used during the development, where I host application images from different registry. I suggest we still retain the `cluster_repo_name` variable, but change the value as `repo_name`, like below
```
cluster_repo_name: "{{ repo_name }}"
```

Reply:
Hi @krsacme thanks for the feedback, I wasn't sure the purpose of having a namespace per cluster.
Your suggestion should help to have a single repo by default but able to override it if needed.
I'll modify the change as you suggested.